.github/workflows/tests.yml

```diff
@@ -67,6 +67,9 @@ jobs:
         # and don't want them to be reinstalled in an "isolated env".
         pip install --no-use-pep517 --no-deps -e .[test,docs]
 
+    - name: Cargo test
```

Comment:
Can we invoke `cargo test` from the python testsuite like we do other external tools (mypy etc)?

Reply:
Sure, we can, as I've written in the very first message. I'm just not sure it makes much sense (i.e. it will look like a single test, etc.)